README.md

```diff
@@ -27,13 +36,39 @@ compatibility. When possible, upgrade your Conjur version to match the
 when using integrations, choose the latest suite release that matches your Conjur version. For any 
 questions, please contact us on [Discourse](https://discuss.cyberarkcommons.org/c/conjur/5).
 
-### From PyPI
+### Supported Services
+
+- Conjur OSS v1.2.0+
+- Conjur Enterprise v5.6.3+
+
+### Sypported Platforms
+
+- macOS Catalina+
+- Windows 10+
+- Red Hat Enterprise Linux 7/8
+
+### Installation
+
+#### Install the CLI
+
+You can access the latest release of the CLI to install on your machine by navigating to our 
+[release](https://github.com/cyberark/conjur-api-python3/releases) page. For instructions on 
+how to setup and configure the CLI, see our official documentation ***(TODO add link to docs)*** 
```

Comment:
Conjur CLI

Reply:
link:
https://docs.conjur.org/Latest/en/Content/Developer/CLI/cli-lp.htm

Reply:
I get "The resource you are looking for has been removed, had its name changed, or is temporarily unavailable."

This will be live when u push?